README.md

```diff
@@ -0,0 +1,175 @@
+# UWFE Firmware
+
+This is the firmware monorepo for the UW Formula Electric team. This repository contains vehicle board firmware, HIL board firmware, and our firmware validation testbed.
+
+# Onboarding Instructions
+
+- [OpenProject Access](#open-project)
+- [SSH Key](#ssh-key-set-up)
+- [Vagrant (Windows Only)](#Vagrant-Environment-Set-Up-for-Windows-Users)
+- [Mac OS](#mac-os-set-up)
+- [Linux](#linux-set-up)
+- [Other Resources](#other-resources)
+
+# Open Project
+
+We use Open Project for task management. (todo: add instructions on how to get access lol)
+
+# SSH Key Set Up
+
+Please see [this page](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) for instructions. Feel free to message in #firmware on Slack with any questions!
```

Comment:
For complete novices, ssh can be slightly confusing so how about the following?

SSH key is used for access credentials (think of it as your username and password). Without it, you can't contribute to the repository.
1. Create a SSH key by following the instructions [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
2. Add the SSH key to your GitHub account by following this [page](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)

Note: You can skip this step if you already have a SSH key configured on GitHub